documentation/debugging.md

```diff
@@ -1,4 +1,88 @@
-# Debugging tests in Leviathan
+# Frequently Asked Questions (FAQ's)
+
+## Table of Contents
+  s- [Checking if Autokit is ready to run tests](#checking-if-autokit-is-ready-to-run-tests)
+  - [Config issues](#config-issues)
+  - [Recovering the Autokit](#recovering-the-autokit)
+  - [How to troubleshoot connection issues between Leviathan and devices under test?](#how-to-troubleshoot-connection-issues-between-leviathan-and-devices-under-test)
+  - [How to write a new test or contribute to an existing test suite?](#how-to-write-a-new-test-or-contribute-to-an-existing-test-suite)
+  - [Debugging tests in Leviathan](#debugging-tests-interactively-in-leviathan)
+
+
+## Checking if Autokit is ready to run tests
+
+Testing with autokit requires several component to work together. It's crucial all steps in the Getting Started guide are completed to make sure the autokit is working correctly. To verify if the setup is correct, you can do the following steps. 
+
+1. To check availablity of a autokit manually, you can run the following command:
+
+```
+$ curl -X POST PUBLIC-DEVICE-URL-OF-AUTOKIT/status
+```
+
```

Comment:
Not sure where is a good place to add this information, but finding the public URL in between several test fleets needs explaining to.